app/views/components/docs/published-dates.yml

```diff
@@ -40,3 +40,9 @@ examples:
       - display_time: 14th October 2000
         note: Updated information on pupil premium reviews and what information schools need to publish on their websites.
         timestamp: 2000-10-14T15:42:37.000+00:00
+  with_custom_margin_bottom:
+    description: |
+      The component accepts a number for margin bottom from 0 to 9 (0px to 60px) using the [GOV.UK Frontend spacing scale](https://design-system.service.gov.uk/styles/spacing/#the-responsive-spacing-scale).
```

Comment:
What margin does the component default to?

Reply:
It defaults to having no `margin-bottom`.

Reply:
Have updated the doc to specify that